### ISSUE TITLE
make all final renaming decisions

### ISSUE THREAD

**Issue by kecnry** [phoebe-project/phoebe2#47]
ecl_alg -> eclipse_method to be more consistent with other parameters (mesh_method, distortion_method)

**Comment by aprsa** [comment]
QUALIFIERS:
ecl_alg -> eclipse_method

period@star -> prot -- deferred to 2.0b
freq@star -> frot   -- deferred to 2.0b

incl@star -> pitch/yaw (or disable for non-devel mode) deferred to 2.0

gravblaw -> gravblaw_bol

alb_refl_bol, alb_lost_bol, alb_heat_bol, alb_scattering_bol -> frac_*
ld_func_bol, ld_coeffs_bol

'pblum' -> 'self'
pbscale -> pblum_ref

maybe for 2.0:
b['pblum@...'] = 12.56
b['pblum@secondary'] = 'primary'

exptime, exposure_method=[none, oversample, convolve], exposure_oversample -> fti_method, fti_oversample

*** implement 'defer=True' for constraints -- deferred to 2.0

ATTRIBUTES:
default_unit -> unit [veto]
relevant_if = 'pbscale:pblum' -> visible_if (?)

TAG NAMES:
context
method -> kind

TAGS:
capital letters for dataset methods -> use *set or plural arrays

'tmpmodel' -> 'latest'

KWARGS:
uncover (in plotting) -> upto
animate('lc01', uncover=True, highlight=True)

*** look up what other packages call the "uncover" functionality

store_mesh -- defer to splitting functionality

'automesh' -> 'mesh' pending on *set change on 'kind'

DATASETS:
name of backends: legacy (wrapper around phoebe1), phoebe (phoebe2)
b.add_compute('legacy') -> compute='legacy01'

CHOICES:
eclipse_method: only_horizon, graham, visible_ratio, visible_partial

visible_ratio -> native

MISC:
_default -> hide

**Comment by kecnry** [comment]
Regarding "uncover" - I couldn't really find many examples from other packages, but did find these:
* matlab uses 'trace' to do something similar to our 'highlight'
* 'uncover' is used in presentation software and beamer to show bullet point transitions

And also thought of a few new possibilities:
* follow
* reveal
* expose

And for completeness, Andrej has suggested these:
* upto
* stop

If anyone has a strong opinion supporting any of these over uncover (except stop, I veto stop), let me know, otherwise I'll just leave it as is.

I've also updated Andrej's comment above just to link to other issues created for deferred items.

**Comment by aprsa** [comment]
intens_weighing should be intens_weighting (unless we mean to imply that we actually weigh the mass of intensities ;)).
